### PR TITLE
[DOCS] Re-add docs for multiple data paths (MDP)

### PR DESCRIPTION
**Pull request by jrodewig**
We deprecated support for multiple data paths (MDP) in 7.13. However,
we won't remove support until after 8.0.

Changes:

* Reverts PR #72267, which removed MDP docs
* Removes a related item from the 8.0 breaking changes.

In the 7.x backports, I'll add some use some `include` statements to reuse the mitigation steps.

### Preview
https://elasticsearch_78342.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/important-settings.html#_multiple_data_paths

### CODE REVIEW

File: docs/reference/migration/migrate_8_0/node.asciidoc [hunk 1]
```diff
@@ -46,103 +46,6 @@ all the corresponding subfolders in parallel. Each node uses the same subfolder
...
---------------------------------------------------
-PUT _cluster/settings
-{
-  "transient": {
-    "cluster.routing.allocation.exclude._name": "target-node-name"
-  }
-}
---------------------------------------------------
-+
-You can use the {ref}/cat-allocation.html[cat allocation API] to track progress
```
Comment (rjernst) [inline docs/reference/migration/migrate_8_0/node.asciidoc:91]: I don't think we want to lose most of this documentation, as it was specifically worked out to help with migration, which users still need to do. Can it be moved somewhere appropriate for that purpose?

Comment (jrodewig) [reply docs/reference/migration/migrate_8_0/node.asciidoc:91]: We'll still have it in the 7.x docs here: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/migrating-7.13.html#breaking_713_infra_core_deprecations

Those docs are also included in any 7.x branch after 7.13.

I can relocate this to a deprecations section in the 8.0 docs as well. That's not standard, but we can make an exception if wanted.

Comment (jrodewig) [reply docs/reference/migration/migrate_8_0/node.asciidoc:91]: Thanks for the discussion @rjernst. I relocated this content over to the path settings page with df2975448e51a33261043a47cd646ab31b3d9e01.

File: docs/reference/setup/important-settings/path-settings.asciidoc [hunk 2]
```diff
@@ -26,3 +26,112 @@ Supported `path.data` and `path.logs` values vary by platform:
...
+include::{es-repo-dir}/tab-widgets/multi-data-path-widget.asciidoc[]
+
+[discrete]
+[[mdp-migrate]]
+==== Migrate from multiple data paths
+
+Support for multiple data paths was deprecated in 7.13 and will be removed
+in a future release.
+
+As an alternative to multiple data paths, you can create a filesystem which
```
Comment (jrodewig) [inline docs/reference/setup/important-settings/path-settings.asciidoc:53]: Everything below this line was copied from the migration guide. The copied text is unchanged.

**Review by rjernst — APPROVED** [review]
LGTM

**Pull request merged**